src/controllers/UsersController.php

```diff
@@ -148,6 +148,16 @@ public function postAction($request, $db){
                 $user_id = $user_mapper->createUser($user);
                 header("Location: " . $request->base . $request->path_info . '/' . $user_id, NULL, 201);
 
+                // autoverify for test platforms
+                if(isset($this->config['features']['allow_auto_verify_users']) 
+                    && $this->config['features']['allow_auto_verify_users']) {
+                        if($request->getParameter("auto_verify_user") == "true") {
```

Comment:
Why a separate if statement rather than an && on the current one.

Also, should auto_verify_user be set to "yes" for consistency with verbose?